### PR TITLE
Add firewatch terms to glossary

### PR DESCRIPTION
**Pull request by misterpantz**
#### Description

add firewatch terms to glossary

#### Related issue

docs backlog

#### Types of changes

- [ ] Functional documentation bug fix (i.e., broken link or some other busted behavior)
- [ ] New functional doc capabilities (i.e., filter search results)
- [x] New content
- [ ] Revision to existing content
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

#### Checklist

- [x] I have read the **README** document about contributing to this repo.
- [x] I have tested my changes locally and there are no issues.
- [x] All commits are signed.

### CODE REVIEW

File: docs/glossary.mdx [hunk 1]
```diff
@@ -21,11 +21,18 @@ An API token gives an application or service access to an application programmin
 
 An asset is a physical or virtual computing, network, or storage device or other component of the information environment. Workstations, servers, SaaS environments, cloud storage, repositories, and virtual machines are only a few examples.
 
+### base score
+
+Each finding that Mondoo reveals has a [base score](/platform/security/posture/findings/#begin-with-a-base-score) associated with it. For misconfigurations exposed by Mondoo's security policies, the base score comes from the policy. For CVEs and advisories, the base score is the CVSS score.
```
Comment (tas50) [inline docs/glossary.mdx:26]: Can CVSS be a link to that item in the glossary?

File: docs/glossary.mdx [hunk 2]
```diff
@@ -21,11 +21,18 @@ An API token gives an application or service access to an application programmin
...
+
+### blast radius
+
+The [blast radius](/platform/security/posture/findings/#blast-radius) of a finding is the impact that the finding has on a space. Mondoo can expose the same finding on multiple assets in a space. It calculates blast radius of the finding using the risk scores of all of the assets in the space that have that finding.
```
Comment (tas50) [inline docs/glossary.mdx:30]: This seems a bit overworded for just saying it's the number of times we've seen this problem

File: docs/glossary.mdx [hunk 3]
```diff
@@ -90,6 +101,14 @@ A [region](/platform/start/organize/regions/) is the part of the world in which
 
 Mondoo's [registry](/platform/security/posture/policies/) is the location in the Mondoo Console where you choose and manage the security policies that are the bases for assessing your infrastructure.
 
+### risk factor
```
Comment (tas50) [inline docs/glossary.mdx:104]: risk factors can lower risk as well like having SELinux configured

File: docs/glossary.mdx (hunk 1 above)
Comment (misterpantz) [reply docs/glossary.mdx]: I did even better!

File: docs/glossary.mdx (hunk 2 above)
Comment (misterpantz) [reply docs/glossary.mdx]: indeed

File: docs/glossary.mdx (hunk 3 above)
Comment (misterpantz) [reply docs/glossary.mdx:104]: added, thanks!

**Review by tas50 — APPROVED** [review]

**Pull request merged**